opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/authorization/UpdateAuthorizationService.java

```diff
@@ -0,0 +1,26 @@
+package de.adorsys.opba.protocol.facade.services.authorization;
+
+import de.adorsys.opba.consentapi.model.generated.InlineResponse200;
+import de.adorsys.opba.protocol.api.UpdateAuthorization;
+import de.adorsys.opba.protocol.api.dto.request.authentication.AuthorizationRequest;
+import de.adorsys.opba.protocol.facade.services.FacadeService;
+import de.adorsys.opba.protocol.facade.services.ProtocolSelector;
+import de.adorsys.opba.protocol.facade.services.ResultHandler;
+import de.adorsys.opba.protocol.facade.services.ServiceContextProvider;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+import static de.adorsys.opba.db.domain.entity.ProtocolAction.UPDATE_AUTHORIZATION;
+
+@Service
+public class UpdateAuthorizationService extends FacadeService<AuthorizationRequest, InlineResponse200, UpdateAuthorization> {
```

Comment:
if we stick here to generated classes from API, like `InlineResponse200`, it's worth to give them names in OpenAPI yaml, so that at least class name won't be changed too often. Generally I'd recommend to use own bean here and map it in the API layer.

Reply:
True, atm. it mostly to make it work